help/deploy/aws/cloudformation/templates/fargate.template.yml

```diff
@@ -1528,20 +1528,72 @@ Resources:
       ServiceRegistries:
         - RegistryArn:
             Fn::GetAtt:
-              - DatagrokJupyterKernelGatewayServiceDiscoveryEntry
+              - DatagrokJupyterServiceDiscoveryEntry
               - Arn
       Tags:
         - Key: datagrok.component
           Value: cvm
         - Key: datagrok.service
-          Value: jupyter_kernel_gateway
+          Value: jupyter
```

Comment:
well, it is jupyter_kernel_gateway in case of AWS, no matter the image

Reply:
should I return `Value: jupyter_kernel_gateway` here?

Reply:
yes